### PR TITLE
Update Spanish translation to be in line with the English one

### PR DESCRIPTION
**Pull request by 1011X**
The missing fields in the Spanish translation are almost completely filled. The only one I didn't include was `terms.body_html` because it seemed like something that should be a separate page that the admin can edit?

Anyways, I used formal language and changed some terms to match a closer Spanish equivalent. There might still be some parts that could be changed, but I'm stopping here so I don't become overwhelmed.

I also took the time to make some minor changes in the English file as recommended by #70.

### CODE REVIEW

**Comment by 1011X** [comment]
You're not a developer until you forget to do what you told someone else to do before.

**Comment by clarfonthey** [comment]
These definitely seem like something to offer upstream, although I'm perfectly fine merging them here. I skimmed over the translations and they seem fine, although I'd prefer that a native speaker take a look at the translations, as there are a lot. I'll give a +1 to this assuming that the other reviewer actually looks over the translations more closely.

**Review by clarfonthey — APPROVED** [review]

**Comment by clarfonthey** [comment]
While the project is still small, we decided to rename the `master` branch to `main`. Unfortunately, you're going to have to reopen this PR on that branch.

File: config/locales/es.yml [hunk 1]
```diff
@@ -222,11 +246,13 @@ es:
       config: Configuración
       feature_deletions: Borrados de cuenta
       feature_invites: Enlaces de invitación
+      feature_profile_directory: Directorio de perfiles
       feature_registrations: Registros
       feature_relay: Relés de federación
```
Comment (scarlet-tobar) [inline config/locales/es.yml:251]: Rara vez se usa "relés" en español. Sugiero usar "relays" para mayor comprensión de los usuarios.

File: config/locales/es.yml [hunk 2]
```diff
@@ -310,7 +361,7 @@ es:
         report: denuncia
       action_taken_by: Acción tomada por
       are_you_sure: "¿Estás seguro?"
-      assign_to_self: Asignármela a mí
+      assign_to_self: Asignár a mí
```
Comment (scarlet-tobar) [inline config/locales/es.yml:364]: "Asignar" no "asignár"

File: config/locales/es.yml [hunk 3]
```diff
@@ -435,14 +518,16 @@ es:
...
+    apply_for_account: Solicitar una invitación
     change_password: Contraseña
-    confirm_email: Confirmar email
+    checkbox_agreement_html: Estoy de acuerdo con las <a href="%{rules_path}" target="_blank">reglas del servidor</a> y los <a href="%{terms_path}" target="_blank">terminos de servicio</a>.
+    confirm_email: Confirmar correo electrónico
     delete_account: Borrar cuenta
     delete_account_html: Si desea eliminar su cuenta, puede <a href="%{path}">proceder aquí</a>. Será pedido de una confirmación.
     didnt_get_confirmation: "¿No recibió el correo de confirmación?"
-    forgot_password: "¿Olvidaste tu contraseña?"
+    forgot_password: "¿Olvidaste su contraseña?"
```
Comment (scarlet-tobar) [inline config/locales/es.yml:530]: "¿Olvidó su contraseña?" es más formal.

File: config/locales/es.yml [hunk 4]
```diff
@@ -501,25 +598,34 @@ es:
...
     mutes: Tienes en silencio
     storage: Almacenamiento
+  featured_tags:
+    add_new: Añadir nuevo
+    errors:
+      limit: Ya ha exhibido la cantidad máxima de etiquetas
   filters:
     contexts:
-      home: Timeline propio
+      home: Cronología propio
```
Comment (scarlet-tobar) [inline config/locales/es.yml:626]: "propia" no "propio"

File: config/locales/es.yml [hunk 5]
```diff
@@ -501,25 +598,34 @@ es:
...
       notifications: Notificaciones
-      public: Timeline público
+      public: Cronología público
```
Comment (scarlet-tobar) [inline config/locales/es.yml:628]: "Pública", no "público"

File: config/locales/es.yml [hunk 6]
```diff
@@ -572,7 +706,7 @@ es:
     table:
       expires_at: Expira
       uses: Usos
-    title: Invitar a gente
+    title: Invitar a la gente
```
Comment (scarlet-tobar) [inline config/locales/es.yml:709]: "Invitar gente" suena más natural.

File: config/locales/es.yml [hunk 7]
```diff
@@ -533,23 +639,51 @@ es:
...
+    status: Estado de verificación
+    view_proof: Ver prueba
   imports:
-    preface: Puedes importar ciertos datos, como todas las personas que estás siguiendo o bloqueando en tu cuenta en esta instancia, desde archivos exportados de otra instancia.
+    modes:
+      merge: Unir
+      merge_long: Mantener registros actuales y agregar nuevos
+      overwrite: Sobrescribir
+      overwrite_long: Reemplazar registros actuales con los nuevos
+    preface: Puedes importar ciertos datos, como todas las personas que está siguiendo o bloqueando en su cuenta en esta instancia, desde archivos exportados de otra instancia.
```
Comment (scarlet-tobar) [inline config/locales/es.yml:679]: "Puede", no "puedes" para que sea más formal.

File: config/locales/es.yml [hunk 8]
```diff
@@ -631,28 +765,66 @@ es:
...
     other: Otros
     publishing: Publicación
     web: Web
+  relationships:
+    activity: Actividad de cuenta
+    dormant: Latente
+    last_active: Última vez activo
+    most_recent: Más reciente
+    moved: Movido
+    mutual: Mutuo
```
Comment (scarlet-tobar) [inline config/locales/es.yml:796]: No se usa "mutuo" como "mutual" en español.

**Review by scarlet-tobar — COMMENTED** [review]
Some comments added to the code (in Spanish). In general it's good, but it has some details.

File: config/locales/es.yml (hunk 6 above)
Comment (1011X) [reply config/locales/es.yml:709]: Cambiado, gracias

File: config/locales/es.yml (hunk 5 above)
Comment (1011X) [reply config/locales/es.yml:628]: Cambiado, gracias

File: config/locales/es.yml (hunk 8 above)
Comment (1011X) [reply config/locales/es.yml:796]: ¿Como se diría aquí, "seguidores comunes"?

File: config/locales/es.yml (hunk 1 above)
Comment (1011X) [reply config/locales/es.yml:251]: Ehh, Google Translate dice que "relé" es muy común en documentos públicos. No creo que sea *tan* raro...

File: config/locales/es.yml (hunk 8 above)
Comment (scarlet-tobar) [reply config/locales/es.yml:796]: Seguidores comunes vendría a ser los seguidores que siguen a ambas personas. No existe traducción directa de "mutual", así que yo opino que se use "mutual" simplemente.

**Pull request closed without merging**